sound/audioclass.py

```diff
@@ -95,34 +55,27 @@ def GetAudioClass(self, data):
         else:
             d.popleft();d.append(self.Decibel)
             self.loop += 1
+        change = data.suddenchange
+        print "sudden sound change value {}".format(data.suddenchange)
+    	
+        loud = '(StateLink Sudden sound change value (NumberNode "' + \
+		       str(change) + '"))'
```

Comment:
there seems to be a mixture of tabs and spaces being used for indentation.  Do not EVER EVER use both tabs and spaces in the same file. Use one, or use the other.  Many people set tab stops to 2 or 3 or 4 instead of 8, and when this is done, the indentation goes crazy.